src/posthog-core.js

```diff
@@ -418,6 +422,7 @@ PostHogLib.prototype._send_request = function (url, data, options, callback) {
                 callback,
                 retriesPerformedSoFar: 0,
                 retryQueue: this._retryQueue,
+                onXHRError: this.get_config('on_xhr_error'),
```

Comment:
This `xhr()` is also called inside `retry-queue.js`. Would it make sense to add this `onXHRError` there as well?